meerk40t/core/node/node.py

```diff
@@ -92,7 +92,7 @@ def __init__(self, *args, **kwargs):
         self._can_update = True
         self._can_remove = True
         self._is_visible = True
-
+        self.mkparam = None
```

Comment:
This seems a little extreme. mkparam for *all* nodes. Maybe a mixin like `Stroked` or something. Most nodes would never utilize this.

Reply:
Agree, that makes sense, although you could extend the concept if you really wanted to images too, but operations do not make a lot of sense